src/views/User/ProfileUpdatePage/ProfileUpdatePage.jsx

```diff
@@ -0,0 +1,197 @@
+                <AvatarWrapper>
+                  <Avatar size="8rem" src={previewURL} />
+                  <StyledSettingsIcon />
+                </AvatarWrapper>
+                <label>
+                  <StyledP>프로필 이미지 변경</StyledP>
+                  <input
+                    type="file"
+                    name="file"
+                    accept=".png,.jpg,.jpeg"
+                    onChange={(event) => {
+                      const file = event.target.files[0];
+                      if (file) setPreviewURL(URL.createObjectURL(file));
+                      setFieldValue('file', file);
+                    }}
+                    hidden
+                  />
+                </label>
+              </ProfilePictureWrapper>
+              <Input
+                type="text"
+                name="nickname"
+                onChange={handleChange}
+                onBlur={handleBlur}
+                value={values.nickname}
+                placeholder="닉네임"
+              />
+              <FormError isVisible={errors.nickname && touched.nickname}>
+                {errors.nickname}
+              </FormError>
+              <Input
+                margin="0 0 4rem 0"
+                type="email"
+                name="email"
+                bgColor="lighterGray"
+                value={values.email}
+                placeholder="이메일"
+                disabled
+              />
+              <Input
+                type="password"
+                name="password"
+                onChange={handleChange}
+                onBlur={handleBlur}
+                value={values.password}
+                placeholder="비밀번호"
+              />
+              <FormError isVisible={errors.password && touched.password}>
+                {errors.password}
+              </FormError>
+              <Input
+                type="password"
+                name="password2"
+                onChange={handleChange}
+                onBlur={handleBlur}
+                value={values.password2}
+                placeholder="비밀번호 확인"
+              />
+              <FormError isVisible={errors.password2 && touched.password2}>
+                {errors.password2}
+              </FormError>
+              <Button
+                type="submit"
+                margin="0 auto"
+                width="26rem"
+                height="4rem"
+                bgColor="normalOrange"
+                onSubmit={handleSubmit}
+                disabled={isSubmitting}>
+                저장하기
+              </Button>
+            </Form>
+          )}
+        </Formik>
+      </ProfileUpdateWrapper>
+    </Wrapper>
+  );
+};
+
+export default ProfileUpdatePage;
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+`;
+const ProfileUpdateWrapper = styled.div`
+  display: flex;
+  margin-top: 5.2rem;
+  width: 100%;
+  height: 90%;
+  justify-content: center;
+  align-content: center;
+`;
+const Form = styled.form`
+  width: 80%;
+`;
```

Comment:
```suggestion
const Wrapper = styled.div`
  padding: 0 2.4rem 2.4rem 2.4rem;
`;

const Form = styled.form`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  margin: 12rem 0 0 0;
`;
```

`ProfileUpdateWrapper`는 없어도 될 것 같아요. div의 기본값 (width 100%)도 활용해주시면 좋을 것 같습니다